.github/workflows/create-release-branch.yaml

```diff
@@ -0,0 +1,28 @@
+name: Create release branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      newVersionNumber:
+        description: 'New version number'
+        required: true
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: fregante/setup-git-user@v1
+
+      - name: Replace version
+        run: |
+          sed -i -e '/version =/ s/= .*/= ${{ github.event.inputs.newVersionNumber }}/' ./plugins/sqlfluff-templater-dbt/setup.cfg
+          sed -i -e '/version =/ s/= .*/= ${{ github.event.inputs.newVersionNumber }}/' ./setup.cfg
+          sed -i -e '/stable_version =/ s/= .*/= ${{ github.event.inputs.newVersionNumber }}/' ./setup.cfg
+
+      - name: Push to feature branch
```

Comment:
It would be great if we could create the section in `CHANGELOG.md` with the version number.

As you say putting in the current changes would be brilliant,  but even just the placeholder section for now and look at that later.

Reply:
Done!